packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -204,7 +204,7 @@ export class AngularCompilerPlugin {
 
     this._rootNames = config.rootNames.concat(...this._singleFileIncludes);
     this._compilerOptions = { ...config.options, ...options.compilerOptions };
-    this._basePath = config.options.basePath || '';
+    this._basePath = config.options.basePath || basePath || '';
```

Comment:
This might change how the plugin works. What was the motivation? I get the feeling it should be either `config.options.basePath || ''` or `basePath || ''` but not the current one.

Reply:
So looking at the code `basePath` itself wasn't used, and there was a lot of logic around it. I also thought it might change behaviour, but it seems that all tests are running fine. I can revert and remove the `basePath` logic entirely.

Reply:
I'm ok with leaving it as is now.